api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -201,6 +200,7 @@ public void validate_shouldFailIfTheStartDatetimeIsAfterAnyEncounter() {
 		visit.setPatient(encounter.getPatient());
 		encounter.setVisit(visit);
 		encounter.setEncounterDatetime(visit.getStartDatetime());
+		Context.flushSession();
```

Comment:
Aren't these lines already added through another ticket?

Reply:
@dkayiwa by the time I had pushed this code,i had not yet made the fix to the stackoverflow failure in the Visit Validator test.